src/events/cd-event-list.vue

```diff
@@ -5,15 +5,19 @@
         <div class="cd-event-list__no-events-header">
           {{ $t('No Upcoming Events') }}
         </div>
-        <p class="cd-event-list__no-events-content" v-html="$t('There are no upcoming events planned for this Dojo. Please email {email} if you have any questions.', { email: '<a href=\'mailto:' + dojo.email + '\'>' + dojo.email + '</a>' })">
-        </p>
+        <p v-if="dojo.private" class="cd-event-list__no-events-content" v-html="$t('There are no upcoming events planned for this Dojo. Please email {email} if you have any questions.', { email: '<a href=\'mailto:' + dojo.email + '\'>' + dojo.email + '</a>' })"></p>
```

Comment:
Splitting the string in 2 sentences would allow to not duplicate some part of the DOM and save some translations work I believe.